Lib/test/test_uuid.py

```diff
@@ -681,6 +683,29 @@ def test_uuid5(self):
             equal(u, self.uuid.UUID(v))
             equal(str(u), v)
 
+    def test_uuid8(self):
+        equal = self.assertEqual
+        u = self.uuid.uuid8()
```

Comment:
Can you add a test creating two `uuid8()` (without arguments, so random) and just check that they are unique (different)?